gtk3/sample/misc/properties.rb

```diff
@@ -1,23 +1,22 @@
 #!/usr/bin/env ruby
 =begin
-  properties.rb - 
+  properties.rb -
 
-  GLib::Object#properties and Gtk::Container#child_properties sample script.
+  GLib::Object#properties and Gtk::Container#list_child_properties sample script.
```

Comment:
Oh, it's an unexpected API change.
I've renamed `Gtk::Container.list_child_properties` to `Gtk::Container.child_properties`. Could you use `child_properties`?